api/arceos_posix_api/Cargo.toml

```diff
@@ -19,11 +19,12 @@ documentation = "https://arceos-org.github.io/arceos/arceos_posix_api/index.html
 [features]
 default = []
 
+uspace = ["axns/thread-local", "axfs/thread-local", "smp", "irq", "fs", "multitask", "net", "pipe", "select", "epoll"]
```

Comment:
We only use `uspace = ["axns/thread-local"]` here, other features should be enabled in starry.